resources/definitions/ubuild-3d_mr_bot_280.def.json

```diff
@@ -0,0 +1,49 @@
+{
+    "id": "ubuild-3d_mr_bot_280",
+    "version": 2,
+    "name": "uBuild-3D Mr Bot 280",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "uBuild-3D",
+        "manufacturer": "uBuild-3D",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "icon": "icon_uBuild-3D",
+        "platform": "mr_bot_280_platform.stl",
```

Comment:
I don't see this STL file in the pull request, please add it in the resources/meshes folder.

Reply:
The icon line can be removed too. It is not used, but if it were used it would refer to an icon file which does not exist.